README.md

```diff
@@ -1,33 +1,123 @@
+and at the [issues](https://github.com/chaoss/wg-gmd/issues) and
+[pull requests](https://github.com/chaoss/wg-gmd/pulls) in
+[this repository](https://github.com/chaoss/wg-gmd).
+
+Then, of course you can participate in the mailing list,
+in online meetings, and in issues and pull requests.
+
+Currently, main lines of work are:
+
+* [Use cases](use_cases). Proposal and discussion of use cases that help to understand metrics in context.
+You can propose your use cases, and/or contribute to refine those that have been proposed already.
+
+* Focus areas (definitions, goals, questions). Currently, in the [definition of GMD metrics] we strucuture
+it in the following focus areas: [Issue Resolution](focus_areas/issue_resolution.md), [Code Development](focus_areas/code_development.md), and [Community Growth](focus_areas/communitiy_growth.md).
+For each area of interest, we're following the [goal-question-metric methodology](https://en.wikipedia.org/wiki/GQM) defining questions and metrics that help to answer them.
+You can contribute by proposing new goals for a focus area, or new questions for learning about those goals,
+or new metrics for answering those questions. Or by helping to refine goals, and questions
+(see about metrics below).
+
+* Metrics definition. For each metric, we have a document describing it,
+all of them in the [metrics directory](metrics).
+You can contribute by helping to refine those metrics definitions.
+
+* Reference implementations. For each metric, we intend to produce reference implementations,
+in the [implementations directory](implementations).
+They are based on data from real data sources,
+retrieved using [Perceval](https://github.com/chaoss/grimirelab-perceval).
+The idea is to first use a Python notebook to study how to produce the metric, and
+all the variations, parameters, etc, that are convenient to have into account.
+And then, produce a simple script that will compute the metric from a Perceval dump.
+These scripts would be used a reference implementations, both for informing other implementations,
+and for ensuring that, if they intend to implement CHAOSS metrics, they produce the same results
+on the same data sources.
+
+In any of these subjects, you can propose your ideas by opening an issue,
+proposing a pull request, introducing your concerns during a GMD meeting,
+or via a message to the mailing list. However, the usual procedure
+(meetings and general comments in the mailing list) is as follows:
+
+* If you think something should be done (including a contribution by yourself),
+please open an issue in this repository. That will allow others to learn that
+you think some work should be done, and can comment on that.
+If you intend to do the job yourself, please say that.
+
+* Everyone with an opinion on the matter should comment on the issue,
+explaining how they support the idea, propose some change to it,
+or think it is not worth / it is not the moment for doing it.
 
-**Next meetings**:
```

Comment:
I am moving this meeting information into a new file.  I will do that pull request now.